comp/process/status/statusimpl/status_test.go

```diff
@@ -77,11 +72,7 @@ func TestStatus(t *testing.T) {
 
 			assert.NoError(t, err)
 
-			// We replace windows line break by linux so the tests pass on every OS
-			expected := strings.Replace(string(textResponse), "\r\n", "\n", -1)
-			output := strings.Replace(b.String(), "\r\n", "\n", -1)
-
-			assert.Equal(t, expected, output)
+			assert.True(t, strings.Contains(b.String(), "API Key ending with:"))
```

Comment:
Rather than testing the full output as before. We can only control the expvar output for now. This test check that the expvar section of the text template is render correctly